### DISCUSSION TITLE
Tracking the Dyanamics of Mpox Genomic Lineages Over the Years

### DISCUSSION THREAD

**Discussion by karthi-16** [has2k1/plotnine#851] [2024 Plotnine Contest]
### Authors

Karthikeyan Govindan

### Links

[github](https://github.com/karthi-16), [linkedin](linkedin.com/in/karthik-govind-5a7433104)

### Full description

This area plot, created using plotnine in Python, predicts the spread of genomic lineages over time. Such plots are valuable for forecasting disease spread, investigating outbreaks, assessing vaccine efficacy, and understanding disease dynamics.
![plot(6)](https://github.com/user-attachments/assets/2d5173e7-d1ad-45fd-991a-bd4506da4f8e)
This plot clearly illustrates the emergence of new lineages during the 2021 mpox outbreak with  rise in their overall percentages.
Such plots are highly valuable for disease outbreak and monitoring analysis as they provide clear visualizations of how new lineages or strains evolve and proliferate over time. By depicting the rise in percentages of different lineages, these plots help researchers and public health officials identify trends, assess the spread of new variants, and develop targeted strategies for containment and response.
The code to reproduce the plot is provided in the Colab notebook, along with the data used. For similar analyses, please import your own data. 
The first part of the code formats the data into a long format suitable for use with plotnine (ggplot2)
This plotting code in plotnine includes:
geom_area(alpha=0.9): Creates an area plot with high opacity.
scale_fill_manual(values=colors): Sets custom colors for the fill.
theme_linedraw(): Applies a clean, minimalist theme.
labs(...): Adds titles, labels, and captions.
theme(...): Customizes text appearance (axis labels, title, caption).
xlab and ylab: Label the x and y axes.

### Code repository

https://github.com/karthi-16/Mpox_epidemiology.git

**Comment by has2k1** [comment]
![](https://private-user-images.githubusercontent.com/106297263/352246069-2d5173e7-d1ad-45fd-991a-bd4506da4f8e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNzE2NTAsIm5iZiI6MTcyMjI3MTM1MCwicGF0aCI6Ii8xMDYyOTcyNjMvMzUyMjQ2MDY5LTJkNTE3M2U3LWQxYWQtNDVmZC05OTFhLWJkNDUwNmRhNGY4ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOVQxNjQyMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YTM2NWQ4YTViZDYyZjIyYmFlY2IxMzgwYzE3NTI5NTMwNWQ4NTAyNjZmNjc1ZDI2MTllNWJhNjgwYjA4YTAwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.5TVKVZGr-CEpDOS3G81cSHGjq5ufjPR03dXNy3vmADQ)

**Comment by EconomiCurtis** [comment]
Ahoy @karthi-16 , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@karthi-16, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
